2022/07.md

```diff
@@ -104,6 +104,10 @@ Supporting materials includes slides, a link to the proposal repository, a link
     |   |     0 |     30m | [Policy Maps & Sets](https://github.com/js-choi/proposal-policy-map-set) for Stage 1 ([slides](https://docs.google.com/presentation/d/1ykMq2uQZbvtawN5MpBMD2Y1PXk3DZEI_O5U0Fsf-_ZQ/edit?usp=sharing)) | J. S. Choi & Hemanth HM |
     |   |     0 |     30m | [Function memoization](https://github.com/js-choi/proposal-function-memo) for Stage 1 ([slides](https://docs.google.com/presentation/d/1yeMIUYnu17QCvIADQ6rGg0v1tCC93EOkMhR66IKbYUU/edit?usp=sharing)) | J. S. Choi & Hemanth HM |
     |   |     0 |     30m | [Demethodize Function](https://github.com/js-choi/proposal-function-demethodize) for Stage 1 (slides soon) | J. S. Choi |
+    |   |     0 |     30m | [Function.pipe & flow](https://github.com/js-choi/proposal-function-pipe-flow) for Stage 1 (slides soon) | J. S. Choi |
+    |   |     0 |     30m | [Math.clamp](https://github.com/js-choi/proposal-math-clamp) for Stage 1 (slides soon) | J. S. Choi |
```

Comment:
It's already on stage 1 - https://github.com/rwaldron/proposal-math-extensions - maybe just return the existing proposal?